mvt/feature.go

```diff
@@ -441,7 +445,7 @@ func (c *cursor) scalePolygon(g tegola.Polygon) (p basic.Polygon) {
 		if len(ln) < 2 {
 			if debug {
 				// skip lines that have been reduced to less then 2 points.
-				log.Println("Skipping line 2", lines[i], len(ln))
+				log.Debug("Skipping line 2", lines[i], len(ln))
```

Comment:
Just want to confirm; we decided that log message should be lowercase. Correct? So this should be:
```
log.Debug("skipping line 2", lines[i], len(ln))
```